numpy/ma/core.py

```diff
@@ -8359,6 +8359,17 @@ def convolve(a, v, mode='full', propagate_mask=True):
     See Also
     --------
     numpy.convolve : Equivalent function in the top-level NumPy module.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from numpy.ma import masked_equal, convolve
+    >>> a = masked_equal(np.arange(5), 2)
+    >>> b = np.array([1, 1])
+    >>> convolve(a, b)
```

Comment:
Sorry for the late comment.  This looks fine, I think most of the code just uses `np.ma.<function>` without explicit import, so maybe that would be clearer?

One other suggestion I have would be to print out `a` (just with `>>> a`), since I am not sure that it is easy for users to grasp without seeing it.